_publications/2022-07-26-portmaster-vs-glasswire.md

```diff
@@ -0,0 +1,208 @@
+
+### Portmaster and GlassWire Provide a Simple Dashboard for Detecting and Responding to Privacy Threats
+
+As far as network monitoring is concerned, the dashboard is an important part that will allow you to understand the network activity of your device. You would want to know which domains/websites were allowed to connect and which were blocked while you enjoyed your memes. In this case, Portmaster and GlassWire take a similar approach. They display all connections on a simple and easy to understand dashboard. GlassWire uses graphs, while Portmaster uses numbers. These connections are also sorted per app to help you differentiate connections made by your browser from the connections made by your chat app.
+
+When it comes to connection details, both Portmaster and GlassWire show the type of traffic, the IP address and country of the connected servers, the time taken to make a connection and similar metrics.
+
+GlassWire as well as Portmaster allow you to better understand your network activity by displaying the past and present connections of apps on the dashboard. GlassWire can display past network activity for up to a month on the free plan. Portmaster currently cannot show a long connection history, but it is on Portmaster's roadmap.
+
+### Portmaster and GlassWire Allow You to Configure Settings System-Wide and Per-App
+
+Portmaster has global settings which define behavior for the whole device. But sometimes, an application may break due to a blocked connection. Allowing that connection system-wide might hurt your privacy. With Portmaster, you can easily solve this problem by creating an exception for a specific application, leaving other apps unaffected.
+
+You can even block risky connection types system-wide, such as p2p or incoming, and then create exceptions for trusted apps.
+
+### GlassWire Seems to Be A Proxy for the Windows Filtering Platform
+
+Disclaimer: GlassWire is closed-source - therefore we had to analyze their software in a blackbox test. We do not have the full picture on how GlassWire integrates into the operating system.
+
+What we did find out is that GlassWire uses the Windows Filtering Platform (WFP) in user-space mode, creating rules for applications essentially exactly like the Windows Firewall does itself. This is a similar approach to that of [Simplewall]({{ site.simplewall_comparison_url }}). When Windows Firewall is disabled, this also makes GlassWire blind and incapable of blocking anything.
+
+GlassWire also uses a background service to collect additional data about connections. It is hard to find out what this service exactly does.
+
+Finally, an oddity we came across was that GlassWire provides a tray icon with the "Exit" option. But pressing that does not end GlassWire or the service as expected - instead the software continues to do what it does. We feel that this is unfavorable wording and might not align with user expectation.
```

Comment:
What I think it does that "Exit" stops the blocking of apps, but the collection of data continues. Sorry if that was not clear from the initial analysis.